### PR TITLE
Create individual AR storybook and chromatic

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

This is the third and final PR for creating individual storybooks, following on from #6590  and #6561

Details:

- add chromatic config for AR (we'll need to add a required check for `UI tests: APPS_RENDERING`
- adds individual storybook config for AR
- composes AR storybook into composed storybooks
- removes shared storybook config
- adds missing dependencies
- sorts out storybook scripts

### CODE REVIEW

**Review by georgeblahblah — COMMENTED** [review]
Thanks for this Joe!

I had one comment about what I think is a missing `mobile` breakpoint.

Additionally, I'm not 100% sure that the rendering of components in storybook is now like-for-like with the existing setup on `main`. I feel it's some combination of fonts/styling causing the differences, but I'm very happy to help investigate further.

## Examples

### Newsletter signup

| Storybook on `main` | Storybook on this branch |
|--------|-------|
|   |    |

### Footer

| Storybook on `main` | Storybook on this branch |
|--------|-------|
|   |    |

### Rich link

| Storybook on `main` | Storybook on this branch |
|--------|-------|
|   |    |

**Comment by georgeblahblah** [comment]
I think we could also delete `apps-rendering/.storybook/mocks/log4js.js`!

**Comment by georgeblahblah** [comment]
Following up on the presentational diff with this storybook, it looks like it's being caused by missing fonts. `preview-head` loads fonts from an `/assets` path which is missing in storybook.

The fonts in DCR's storybook are loaded from `assets.guim` and we should probably look at doing the same for AR's storybook. Ideally AR would also load fonts from `assets.guim`, as captured in https://github.com/guardian/dotcom-rendering/issues/4486.

File: apps-rendering/.storybook/preview.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+if (isChromatic()) {
+	// Fix the date to prevent false negatives
+	MockDate.set('Sat Jan 1 2022 12:00:00 GMT+0000 (Greenwich Mean Time)');
+}
+
+setCookie({ name: 'bwid', value: 'mockBrowserId' });
+
+const guardianViewports = {
+	mobileMedium: {
```
[the remark this answers is not in the document]
Comment (joecowton1) [reply apps-rendering/.storybook/preview.js:25]: Thats a great idea re source-foundations - maybe one for another PR?

For now i've just added in the suggestion of `mobile`

File: apps-rendering/.storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	wide: {
+		name: 'wide',
+		styles: {
+			width: '1300px',
+			height: '800px',
+		},
+	},
+};
+
+export const viewports = [375, 480, 660, 740, 980, 1140, 1300];
```
[the remark this answers is not in the document]
Comment (joecowton1) [reply apps-rendering/.storybook/preview.js]: added!

**Comment by joecowton1** [comment]
> Following up on the presentational diff with this storybook, it looks like it's being caused by missing fonts. `preview-head` loads fonts from an `/assets` path which is missing in storybook.
> 
> The fonts in DCR's storybook are loaded from `assets.guim` and we should probably look at doing the same for AR's storybook. Ideally AR would also load fonts from `assets.guim`, as captured in #4486.

I've added a `fonts-css` file similar to our fix for [common-rendering](https://github.com/guardian/dotcom-rendering/pull/6599). Going forward we spoke about using GUIM to host AR fonts permanently. 

I've also added the css reset from source-foundations to fix some styling issues.

**Pull request merged**